fcrepo-http-api/pom.xml

```diff
@@ -77,11 +77,6 @@
       <version>${project.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.fcrepo</groupId>
-      <artifactId>fcrepo-connector-file</artifactId>
-      <version>${project.version}</version>
```

Comment:
It would make _more_ sense. I guess it depends on what view we take of the projection functionality. @awoods, thoughts?